compute/delta.go

```diff
@@ -25,6 +25,17 @@ func (id DeltaID) String() string {
 	return base58.Encode(id)
 }
 
+func (id DeltaID) LessThan(other DeltaID) bool {
```

Comment:
we can use the built-in function to compare to bytes.
byte.Compare(a, b []byte) int

Reply:
We noticed this before but looks like we forgot to do this. Updated.